### PR TITLE
Rename AST nodes for Fluent 0.6

### PR DESCRIPTION
**Pull request by zbraniecki**
Simple renames for #206.

### CODE REVIEW

File: fluent-syntax/src/ast.js [hunk 1]
```diff
@@ -115,10 +115,10 @@ export class MessageReference extends Expression {
   }
 }
 
-export class ExternalArgument extends Expression {
+export class VariableExpression extends Expression {
```
Comment (stasm) [inline fluent-syntax/src/ast.js]: This should be called `VariableReference`.

File: fluent-syntax/src/serializer.js [hunk 2]
```diff
@@ -227,7 +227,7 @@ function serializeMessageReference(expr) {
 }
 
 
-function serializeExternalArgument(expr) {
+function serializeVariableExpression(expr) {
```
Comment (stasm) [inline fluent-syntax/src/serializer.js]: `…VariableReference` here as well.

File: fluent-syntax/src/ast.js (hunk 1 above)
Comment (zbraniecki) [reply fluent-syntax/src/ast.js]: Ahh, great catch! I was following https://github.com/projectfluent/fluent.js/issues/206 which has it as `VariableExpression`. Fixing :)

Comment (stasm) [reply fluent-syntax/src/ast.js]: My bad. I fixed the checklist in #206.

**Pull request merged**